src/menu-list.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const React = require('react');
+const mdl = require('material-design-lite/material.min');
+const classnames = require('classnames');
+
+const baseClasses = {
+  'mdl-menu': true,
+  'mdl-js-menu': true
+};
+
+class MenuList extends React.Component {
+ render(){
+  const {
+    className,
+    children,
+    ripple,
+    topLeft,
+    topRight,
+    bottomRight,
+    htmlFor
+  } = this.props;
+
+  const classes = classnames(baseClasses, {
+    'mdl-js-ripple-effect': ripple,
+    'mdl-menu--top-left': topLeft,
```

Comment:
I'm trying to figure the way to do this without changing the 1 for 1 ideas to date, and I feel like if we're going to restrict this then we should probably just leave it up to the user to put it into className and default to documentation.

we could do <MenuList position='top-left'> but then it just makes more sense for them to utilize the already made <MenuList className='top-left'> because it flows with documentation better.